core/src/main/resources/jenkins/security/QueueItemAuthenticatorMonitor/message.properties

```diff
@@ -12,5 +12,6 @@ queueItemAuthenticatorConfigured = \u2705 Access control for builds is configure
 noQueueItemAuthenticatorConfigured = \u274c Access control for builds is possible, but not configured. Configure it in the <a href="{0}/configureSecurity">global security configuration</a>.
 
 anyBuildLaunchedAsSystem = \u274c Builds have been observed as launching as the internal SYSTEM user despite access control for builds being set up. \
```

Comment:
Eventually, it would be nice if SYSTEM was decorated in the same style as anonymous (see earlier)